vassal-app/src/main/java/VASSAL/build/module/gamepieceimage/FontStyleConfigurer.java

```diff
@@ -89,7 +89,7 @@ protected void buildFonts() {
       fontPanel.remove(fonts);
     }
 
-    fonts = new JComboBox<>();
+    final JComboBox<String> fonts = new JComboBox<>();
```

Comment:
This is wrong. You're replacing assignment to a member with declaration of a new local variable with a name shadowing the member.

Reply:
I thought the double declaration looked weird, but that's what the AI suggested. See, our jobs are safe.

Reply:
I'm not worried.